src/crimson/osd/backfill_state.cc

```diff
@@ -352,9 +352,14 @@ BackfillState::Enqueuing::Enqueuing(my_context ctx)
     logger().debug("{}: reached end for current local chunk",
                    __func__);
     post_event(RequestPrimaryScanning{});
-  } else if (backfill_state().progress_tracker->tracked_objects_completed()) {
-    post_event(RequestDone{});
```

Comment:
I think the reaction for `RequestDone` has been removed from `Enqueuing` in the previous commit but I see the reason for splitting the work between two patches. OK.